news/scripts/mongo_news_db/db_connector.py

```diff
@@ -1,11 +1,13 @@
 from typing import List
-from models.news import Article
-from pymongo import ReplaceOne
+
 import pymongo
+from pymongo import ReplaceOne
```

Comment:
we already import all of pymongo, isn't ReplaceOne included?

Reply:
Didn't really want to change this since it was alvyn's code

Reply:
On his current PR

Reply:
i'll use make use of only `import pymongo` in my changes